doc/api/modules.md

```diff
@@ -63,6 +63,15 @@ module.exports = class Square {
 
 The module system is implemented in the `require('module')` module.
 
+## Enabling
+
+<!-- type=misc -->
+
+Node.js treats JavaScript code as CommonJS modules by default.
+Authors can tell Node.js to treat JavaScript code as CommonJS modules
```

Comment:
This seems weird to say “it’s the default, and here’s how you can make it apply”.

For file extensions, .cjs is always CJS, .mjs is always ESM, .js is CJS by default (but type module can make this be ESM), and anything else is either .json, .node, .wasm, or “unknown” (which is treated as CJS). Piped input is CJS by default, but can be made ESM with the input-type flag.

Can we say something basically exactly like that?

Reply:
I'll remove this section from this PR and move it to a separate PR so it can be discussed separately if that's ok.